code/modules/mob/living/living.dm

```diff
@@ -600,6 +601,61 @@
 		return 1
 	return 0
 
+/mob/living/proc/makeTrail(turf/T)
+	if(!has_gravity(src))
+		return
+	var/blood_exists = 0
+	var/mob/living/carbon/human/H = src
+
+	if(H.species.exotic_blood)
```

Comment:
Put this in human/MakeTrail so that if there's species exotic blood, it returns, just like with NOBLOOD